### ISSUE TITLE
Request cart dialog element should have aria label

### ISSUE THREAD

**Issue by eliotjordan** [pulibrary/pulfalight#1790]
refs #1717 

### Location of problem on site

Request cart

### WCAG success criterion

[[1.3.1 Info and Relationships (Level A)](https://webaim.org/standards/wcag/checklist#sc1.3.1)](https://www.w3.org/WAI/ARIA/apg/patterns/dialog-modal/)

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Roles/dialog_role

### Recommended fix

Add an aria label to the dialog element.
